pyproject.toml

```diff
@@ -24,5 +24,19 @@ dependencies = [
   'uvloop == 0.16.0; platform_machine == "x86_64"',
   'uvloop == 0.16.0; platform_machine == "i686"'
 ]
+classifiers = [
```

Comment:
Hey @Kludex ,

I hope everything is well. 

I did add the Python Classifiers(taking inspiration from uvicorn and the pypi examples).

Can you have a look if you have anything to recommend/change in these?